webapp/src/components/sidebar_buttons/sidebar_buttons.jsx

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {Tooltip, OverlayTrigger} from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import {makeStyleFromTheme, changeOpacity} from 'mattermost-redux/utils/theme_utils';
+
+import {RHSStates} from '../../constants';
+
+export default class SidebarButtons extends React.PureComponent {
+    static propTypes = {
+        theme: PropTypes.object.isRequired,
+        isTeamSidebar: PropTypes.bool,
+        showRHSPlugin: PropTypes.func.isRequired,
+        issues: PropTypes.arrayOf(PropTypes.object),
+        inIssues: PropTypes.arrayOf(PropTypes.object),
+        outIssues: PropTypes.arrayOf(PropTypes.object),
+        actions: PropTypes.shape({
+            list: PropTypes.func.isRequired,
+            updateRhsState: PropTypes.func.isRequired,
+        }).isRequired,
+    };
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            refreshing: false,
+        };
+    }
+
+    componentDidUpdate() {
+        this.getData();
+    }
+
+    getData = async (e) => {
+        if (this.state.refreshing) {
+            return;
+        }
+
+        if (e) {
+            e.preventDefault();
+        }
+
+        this.setState({refreshing: true});
+        await Promise.all([
+            this.props.actions.list(false, 'my'),
+            this.props.actions.list(false, 'in'),
+            this.props.actions.list(false, 'out'),
+        ]);
+        this.setState({refreshing: false});
+    }
+
+    openRHS = (rhsState) => {
+        this.props.actions.updateRhsState(rhsState);
+        this.props.showRHSPlugin();
+    }
+
+    render() {
+        const style = getStyle(this.props.theme);
+        const isTeamSidebar = this.props.isTeamSidebar;
+
+        let container = style.containerHeader;
+        let button = style.buttonHeader;
+        let placement = 'bottom';
+        if (isTeamSidebar) {
+            placement = 'right';
+            button = style.buttonTeam;
+            container = style.containerTeam;
+        }
+
+        const issues = this.props.issues || [];
+        const inIssues = this.props.inIssues || [];
+        const outIssues = this.props.outIssues || [];
+        const refreshClass = this.state.refreshing ? ' fa-spin' : '';
+
+        return (
+            <div style={container}>
+                <OverlayTrigger
+                    key='todoLink'
+                    placement={placement}
+                    overlay={<Tooltip id='todoTooltip'>{'Your todos'}</Tooltip>}
```

Comment:
We try to use on all the user facing strings the term "Todo" (or "Todos" in this case), that is, with the first letter uppercase. Could you correct it here?